src/plugins/hotkeyPlugin.ts

```diff
@@ -92,20 +77,17 @@ const isNotSameHotkeyTarget = (a: HotkeyTarget) => (b: HotkeyTarget) => {
 export class HotkeyManager {
   /** 登録されたHotkeyAction */
   private actions: HotkeyAction[] = [];
+  /** スコープ */
+  private scope: EditorType = "talk";
```

Comment:
あ、初期値は結構ミスの元になるんですよね。
ここはソング側で設定しないといけなかった時に設定し忘れたとしても気づけないので、初期値無い方が良いかなと！
```suggestion
  private scope: EditorType;
```